quranHome/src/main/res/layout/quran_home_view.xml

```diff
@@ -36,30 +22,43 @@
         app:layout_constraintBottom_toBottomOf="@id/tabLayout"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent">
```

Comment:
here i just used CardView for radius based on UI
is there any view for this purpose ?
I think about simple with a custom shape but i don't think it was smooth when animated.

Reply:
You mean, in case of animating the radius of CardView, is not smoothly enough?

Reply:
no , cardView can remove in this case because we just need radius.
I think i must create custom view and handle radius and use simpler view than cardView

Reply:
YAGNI yet 😉

Reply:
fixed 😍